### DISCUSSION TITLE
requests from UW cataloger

### DISCUSSION THREAD

**Discussion by gerontakos** [uwlib-cams/uwlswd_ngl#22] [requested_updates]
# request details
## Methodist newspapers
* UW cataloger LP used the term in March 2023 (as advised by @gerontakos , despite the fact it is not enumerated in the thesaurus) and wrote the following: "It would be great if “Methodist newspapers” could be added to the list.  Then for sure I could use the newspaper list source code (ngl) in the heading on OCLC.  Without hearing from @gerontakos about what exactly is meant by “etc.”, I’m not sure I should/can include the ngl source code in the heading without having the term added to the list.  I can always add a genre heading for “Methodist newspapers” to the OCLC record without including a source code, but it would be better, more authoritative-looking, to have it."
* LP also wrote to JA: "I’m cataloging the microfilm for 'The Seattle Methodist' and would like to use 'Methodist newspapers' as a genre term, but it isn’t on the Newspaper Genre List. The list has this instruction for religious newspapers:
    * "Religious denomination newspapers
 	* "USE specific descriptors, e.g. Baptist newspapers, Catholic newspapers, Non-denominational newspapers, etc.
    * "The list says to contact you with suggestions and questions.  If it isn’t too much trouble, can you add 'Methodist newspapers' to the list?  Or does the 'etc.' in the instruction mean I can use the term without having it officially added to the list but still say the source is the Newspaper Genre List when I input the genre heading in the OCLC record?  Then you wouldn’t have to take the time to add it to the list.  (Hope this makes sense!)
* JA responded: "Yes, this makes sense!  I’m in favor of you using 'Methodist newspapers.' I’m cc’ing @gerontakos and @E-Fl0 (an iSchool student working in GMM) because they are working with the genre list.  Please chime in if we’re misinterpreting the 'etc.'" 

## Tulalip newspapers
* LP used the term in March 2023 (as advised by @gerontakos , despite the fact it is not enumerated in the thesaurus) and wrote the following:  "...while we’re on subject of potentially adding headings to the Newspaper Genre List, is it possible to add one for Tulalip newspapers?  Last week I was working on a Tulalip Tribe newspaper for Special Collections; there currently isn’t a heading for Tulalip newspapers, but I found a similar generic instruction for Tribal newspapers as for Religious denomination newspapers:
    * "USE descriptors that identify tribal community or communities for whom the newspaper is published, e.g. Cheyenne newspapers, Lummi newspapers. Assign also the descriptor Native American newspapers
* "Since there wasn’t a heading for Tulalip newspapers, I just used Native American newspapers—Washington (State), but I’d really like to also use the more specific term if possible."

**Comment by briesenberg07** [comment]
Tagging @AdamSchiff FYI
These requests, and those recorded in #23 , are the current outstanding requests.
As mentioned previously, I don't believe that any of these changes have been made yet.
